### ISSUE TITLE
M + NUM not working.

### ISSUE THREAD

**Issue by CoolJoey244** [elix-x/toolscompressor#2]
I am pressing M + 6 and all the other stuff, even tried to change the control. It just reverts to the combined tool. I can't even use by bow! Please help. Thank you.

**Comment by Elix-x** [comment]
Try pressing it in another order (Numpad 6 and then M (be sure, that when you press M, 6 is pressed)

**Comment by Elix-x** [comment]
@CoolJoey244 
Did it work?

**Comment by CoolJoey244** [comment]
i haven't tried yet. I just saw you're message, i will by 5:00 tomorrow.

**Comment by CoolJoey244** [comment]
Okay,,, still doesn't work. Do you have any more ideas?

**Comment by Elix-x** [comment]
Not really. For me it works. Try this: hold down 6 (numpad) and press M (or what ever key you set)

**Comment by CoolJoey244** [comment]
It is not working for me either. I did try that too.

**Comment by Elix-x** [comment]
This is very very very starnge... Nave you set numpad 6 for something other or not?

**Comment by CoolJoey244** [comment]
No I haven't. Do you want to know my specs of my PC?

**Comment by CoolJoey244** [comment]
Okay,,, this is weird. Everything works, sorry for complaining, but there is one problem, It only works if you press the NUM + # fast, like tapping a phone. can you make it so when you hold it, it does it?

**Comment by Elix-x** [comment]
sorry for delay, i've been working with HyperMan19680.
Actually code is written like so:
when M is down, check:
NUMPAD 0 is down? -No
NUMPAD 1 is down? -No
NUMPAD 2 is down? -No
NUMPAD 3 is down? -No
NUMPAD 4 is down? -No
NUMPAD 5 is down? -No
NUMPAD 6 is down? -Yes
NUMPAD 7 is down? -No
NUMPAD 8 is down? -No
NUMPAD 9 is down? -No

send Packet to server: M is down and NUMPAD 6 is down.

That means: press M, while holding down numpad 6!

**Comment by HyperMan19680** [comment]
Honestly I'd like to see them both work. And maybe a way to tell when the tool is locked.

**Comment by Elix-x** [comment]
@HyperMan19680 
There's a way to tell, that tool is blocked: look in description (Current mode is (not) fixed)

**Comment by HyperMan19680** [comment]
Ah, I see. I'll check once I get home. I should mention I am seeing some sort of debug text in the tool description...

**Comment by CoolJoey244** [comment]
Sorry again for the delay on replys, but I have to ask you this: did you just update the mod? I have 1.0.0v.
